AzureResourceManager/scripts/install_artifactory.sh

```diff
@@ -209,6 +209,20 @@ cat <<EOF >/var/opt/jfrog/artifactory/etc/binarystore.xml
 </config>
 EOF
 
+# callhome metadata
+
+mkdir -p /var/opt/jfrog/artifactory/etc/info
+cat <<EOF >/var/opt/jfrog/artifactory/etc/info/installer-info.json
+{
+  "productId": "JFrogInstaller_Azure_SQL/1.0.0"
+  "features": [
+  {
+    "featureId": "Artifactory/${artifactoryVersion}",
+    "attributes": {}
```

Comment:
Since attributes are optional, you can skip the empty attributes setting

Reply:
line removed